### PR TITLE
Update 60-sensor.hwdb for Digibras F10-30 rotation info

### PR DESCRIPTION
**Pull request by feliperodriguesfr**
Added Digibras F10-30 rotation info to hwdb.d/60-sensor.hwdb.

### CODE REVIEW

**Review by bluca — APPROVED** [review]
LGTM but would be great to have an ack from the domain experts

**Review by hadess — APPROVED** [review]
I don't particularly like the commit message used, but commit content looks fine.

File: hwdb.d/60-sensor.hwdb [hunk 1]
```diff
@@ -307,6 +307,14 @@ sensor:modalias:acpi:INVN6500*:dmi:*svnDell*:pnVenue10Pro5055:*
 sensor:modalias:acpi:SMO8500*:dmi:*svn*DEXP*:*pn*DEXPOEM:*
  ACCEL_MOUNT_MATRIX=0, -1, 0; -1, 0, 0; 0, 0, 1
 
+#########################################
+# Digibras
+#########################################
+
+# Digibras F10-30
+sensor:modalias:acpi:SMO8500*:dmi:*:svnDigibras:pnF10-30:*
+  ACCEL_MOUNT_MATRIX=-1, 0, 0; 0, 1, 0; 0, 0, 1
```
Comment (yuwata) [inline hwdb.d/60-sensor.hwdb]: Please use only one white space before `ACCEL_MOUNT_MATRIX=`.

**Review by yuwata — CHANGES_REQUESTED** [review]

**Comment by yuwata** [comment]
> I don't particularly like the commit message used, but commit content looks fine.

@feliperodriguesfr Please also update the commit message something like `hwdb: add Digibras F10-30 rotation information`.

**Pull request merged**